datacats/environment.py

```diff
@@ -219,10 +222,20 @@ def load(cls, environment_name=None, data_only=False):
         except NoOptionError:
             always_prod = False
         try:
-            deploy_target = cp.get('deploy', 'target', None)
-        except NoSectionError:
-            deploy_target = None
+            deploy_target = cp.get(
+                'deploy', 'remote_server_user', None) \
+                + "@" + cp.get('deploy', 'remote_server', None)
+        except NoOptionError:
```

Comment:
Should also except NoSectionError, as a lack of a [deploy] section is the default behaviour.